src/configuration/webpack.development.js

```diff
@@ -43,7 +43,16 @@ module.exports = {
                 enforce: 'pre',
                 test: /\.js$/,
                 exclude: /node_modules/,
-                loader: 'eslint-loader'
+                loader: 'eslint-loader',
+                options: {
+                    configFile: resolve(
+                        process.cwd(),
+                        'node_modules',
+                        'cnn-birdman',
+                        'src',
+                        '.eslintrc'
+                    )
+                }
             },
```

Comment:
since `eslint-loader` allows you to use `eslint` settings - i think we should probably switch this to [use baseConfig and disable use of `.eslintrc`](http://eslint.org/docs/developer-guide/nodejs-api#cliengine):
```js
baseConfig: {
   extends: [ require.resolve('eslint-config-cnn-starter-app') ],
},
useEslintrc: false,
```